manifests/init.pp

```diff
@@ -54,6 +54,23 @@
 #
 # $repo_gpgkey::        The GPG key to use
 #
+# $candlepin_db_host::  Host with Candlepin DB
+#
+# $candlepin_db_port::  Port accepting connections to Candlepin DB
+#
+# $candlepin_db_name::  Name of the Candlepin DB
+#
+# $candlepin_db_user::  Candlepin DB user
+#
+# $candlepin_db_password:: Candlepin DB password
+#
+# $candlepin_db_ssl::   Boolean indicating if the connection to the database should be over
```

Comment:
What about adding "Requires DB host's CA Cert in the system trust" or something similar ?